packages/block/package.json

```diff
@@ -5,8 +5,10 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "files": [
-    "dist"
+    "dist",
+    "dist.browser"
```

Comment:
`dist.browser` fields were missing here, added these in 4f33ced9c731737c9485c681704a19c37c2ab25b

Reply:
Ah!!! Good catch!